snapcraft/storeapi/_client.py

```diff
@@ -16,6 +16,12 @@ class Client():
     """
 
     def __init__(self, conf, root_url):
+        """Set up Client object
```

Comment:
Instead of Set up, which is a concept for the test cases, I think it would be better to use the word `Initialize` here. If you search the codebase, we are already using this word for other docstrings.